src/include/catalog/catalog_content.h

```diff
@@ -89,34 +80,18 @@ class CatalogContent {
     const Property& getRelProperty(
         common::table_id_t tableID, const std::string& propertyName) const;
 
-    inline const std::vector<Property>& getNodeProperties(common::table_id_t tableID) const {
-        return nodeTableSchemas.at(tableID)->getProperties();
-    }
-    inline const std::vector<Property>& getRelProperties(common::table_id_t tableID) const {
-        return relTableSchemas.at(tableID)->getProperties();
-    }
-    inline std::vector<common::table_id_t> getNodeTableIDs() const {
-        std::vector<common::table_id_t> nodeTableIDs;
-        for (auto& [tableID, _] : nodeTableSchemas) {
-            nodeTableIDs.push_back(tableID);
-        }
-        return nodeTableIDs;
+    inline const std::vector<Property*> getAllProperties(common::table_id_t tableID) const {
```

Comment:
I would apply a change on the function naming style across this file.

Rename `getAllXXX` as `getXXX`. `All` doesn't serve any purpose in the name